.circleci/slack-notification.sh

```diff
@@ -15,6 +15,7 @@ if [ "${CIRCLE_BRANCH}" == "master" ] ; then
     author=${author/rschwanhold/<@robert>}
     author=${author/tmbo/<@tmbo>}
     author=${author/valentin-pinkau/<@valentin>}
+    author=${author/youri-k/<@youri>}
```

Comment:
I added myself to the notifications. I hope you're fine with doing this in this pr @fm3